dom.go

```diff
@@ -691,6 +692,10 @@ func (d document) AdoptNode(node Node) Node {
 	return wrapNode(d.Call("adoptNode", node.Underlying()))
 }
 
+func (d document) ImportNode(node Node, deep bool) Node {
```

Comment:
Split this PR into (at least) two commits please. ImportNode and RequestAnimationFrame are unrelated. I'm fine with them being in the same PR, as I'm going to accept both, but they should be independent commits.